### PR TITLE
Time build targets and notify #vic-bots of build status

### LINKED ISSUE #307: send drone results to #vic-bots slack channel
cello team does this and can provide details.

### PR DESCRIPTION
**Pull request by sgairo**
Fixes #307 #748

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -117,7 +117,7 @@ tethers: $(tether-linux) $(tether-windows) $(tether-darwin)
 # utility targets
 goversion:
 	@echo checking go version...
-	@( $(GO) version | grep -q $(GOVERSION) ) || ( echo "Please install $(GOVERSION) (found: $$($(GO) version))" && exit 1 )
+	@($(GO) version | grep -q $(GOVERSION) ) || ( echo "Please install $(GOVERSION) (found: $$($(GO) version))" && exit 1 )
```
Comment (mdubya66) [inline Makefile]: was the space needed?

**Comment by jzt** [comment]
LGTM

File: .drone.yml [hunk 2]
```diff
@@ -96,6 +112,14 @@ build:
...
       success: true
 
+notify:
+  slack:
+    channel: vic-bots
+    webhook_url: $$SLACK_URL
+    username: drone
+    when:
+      repo: vmware/vic
+
```
Comment (andrewtchin) [inline .drone.yml]: maybe add `branch: master`, but not many people branch off vmware/vic so not too big of a deal

**Comment by andrewtchin** [comment]
lgtm

**Comment by mdubya66** [comment]
with the fix to put back the removed space
lgtm

File: .drone.yml (hunk 2 above)
Comment (sgairo) [inline .drone.yml]: fixed

File: Makefile (hunk 1 above)
Comment (sgairo) [inline Makefile]: not sure, but fixed to be safe

**Comment by andrewtchin** [comment]
lgtm

**Pull request merged**